Kickstarter-iOS/Features/Dashboard/Controller/DashboardViewController.swift

```diff
@@ -198,6 +205,34 @@ internal final class DashboardViewController: UITableViewController {
     }
   }
 
+  fileprivate func setupDashboardRemovalWarning() {
+    if #available(iOS 15, *) {
+      if let window = UIApplication.shared.windows.filter({ $0.isKeyWindow }).first,
+        let tabBar = self.tabBarController?.tabBar {
+        dashboardRemovalWarningHostingController = UIHostingController(rootView: DashboardRemovalWarning())
+        /// Added to the window because views added to a tableviewcontroller are automatically scrollable and we need this to be a sticky view
+        window.addSubview((dashboardRemovalWarningHostingController?.view)!)
```

Comment:
calling out my comment here. anything added to a UITableViewController (DasboardViewController), is automatically added as a scrollable object. To get the banner to stick above the tabbar I had to add it to the window directly.

Because of this, its important to remember to remove the view manually on viewWillDisappear which you can see above.

If you know of an alternative here please point it out. thanks!

Reply:
So I made sure not to use the `window`, because its' not really a convention we use in the app to add new views.
You can reference the `self.tabBarController` from within any view controller presented by the `tabBar`.
I also used the `MessageBannerViewController` `configureMessageBannerViewController` method to figure out how to add a temporary view to the existing hierarchy, but instead of anchoring to the bottom of the `tabBarController` `view`, anchored to the `top`.

Well go over it in the call 👍🏿